### PR TITLE
Docs: Installation instructions for SLES

### PR DESCRIPTION
**Pull request by aazam-gh**
Updated `setuphttpd.rst` to include installation instructions for SLES

### CODE REVIEW

File: doc/source/setuphttpd.rst [hunk 1]
```diff
@@ -82,7 +82,38 @@ To change that, you need to modify both Apache configuration and firewall rules.
 SUSE Linux Enterprise Server
 ----------------------------
 
-(to be written).
+The following instructions are for SLES15 SP4.
```
Comment (Bischoff) [inline doc/source/setuphttpd.rst:85]: According to table above, they are for 15.5.

Can you please update online your target distro to 15.5? I can help you with that if needed.

File: doc/source/setuphttpd.rst [hunk 2]
```diff
@@ -82,7 +82,38 @@ To change that, you need to modify both Apache configuration and firewall rules.
...
+
+Install the downloaded packages using the `zypper` command
+
+..code-block:: text
+
+    # zypper install zthin-<version>-<release>.s390x.rpm
+    # zypper install zvmsdk-<version>-<release>.noarch.rpm
+
+
+The python routes package must be manually installed as a workaround for enabling the webserver to work, it should not be needed in future versions.
```
Comment (Bischoff) [inline doc/source/setuphttpd.rst:96]: Please move the workaround instructions *above* the package installation instructions. Reason is, user needs to apply the workaround first, and then install.

"web server" in two words (?).

**Review by Bischoff — COMMENTED** [review]
Very good first draft, thank you.

See the remarks. The update to 15.5 is painful, but we need to do it.

**Comment by bjhuangr** [comment]
@aazam-gh still work in progress?

**Comment by aazam-gh** [comment]
@bjhuangr Yes, will be ready soon after testing on z/VM

File: doc/source/setuphttpd.rst [hunk 3]
```diff
@@ -34,7 +34,7 @@ The last section of this chapter details about using tokens to enhance security.
 Tested                 RHEL 9.4           SLES 15.5         Ubuntu 24.04
```
Comment (mfriesenegger) [inline doc/source/setuphttpd.rst:34]:
```suggestion
Tested                 RHEL 9.4           SLES 15 SP5         Ubuntu 24.04
```

File: doc/source/setuphttpd.rst [hunk 4]
```diff
@@ -82,7 +82,33 @@ To change that, you need to modify both Apache configuration and firewall rules.
...
+The RPM packages can be downloaded from (to be determined).
+
+
+
+Install the downloaded packages using the `zypper` command
+
+..code-block:: text
+
+    # zypper install zthin-<version>-<release>.s390x.rpm
+    # zypper install zvmsdk-<version>-<release>.noarch.rpm
```
Comment (mfriesenegger) [inline doc/source/setuphttpd.rst:95]: Is this needed because the zypper install command is included in the quickstart section?

**Review by mfriesenegger — CHANGES_REQUESTED** [review]

File: doc/source/setuphttpd.rst (hunk 4 above)
Comment (Bischoff) [reply doc/source/setuphttpd.rst:95]: I would prefer to duplicate?

File: doc/source/setuphttpd.rst [hunk 5]
```diff
@@ -34,7 +34,7 @@ The last section of this chapter details about using tokens to enhance security.
 Tested                 RHEL 9.4           SLES 15.5         Ubuntu 24.04
 ====================== ================= ================= =================
 Apache2 + uwsgi        ✓                                   
-Apache2 + mod_wsgi     ✓                                   
+Apache2 + mod_wsgi     ✓                 ✓
```
Comment (Bischoff) [inline doc/source/setuphttpd.rst:37]: Please use three check marks on this line, we now also do Ubuntu with the mod_wsgi module for Apache

File: doc/source/setuphttpd.rst [hunk 6]
```diff
@@ -34,7 +34,7 @@ The last section of this chapter details about using tokens to enhance security.
 Tested                 RHEL 9.4           SLES 15.5         Ubuntu 24.04
 ====================== ================= ================= =================
 Apache2 + uwsgi        ✓                                   
-Apache2 + mod_wsgi     ✓                                   
+Apache2 + mod_wsgi     ✓                 ✓                 
 nginx + uwsgi
```
Comment (Bischoff) [inline doc/source/setuphttpd.rst:38]: Add a check mark in "Ubuntu" column for nginx + uwsgi

File: doc/source/setuphttpd.rst [hunk 7]
```diff
@@ -82,7 +82,33 @@ To change that, you need to modify both Apache configuration and firewall rules.
 SUSE Linux Enterprise Server
 ----------------------------
 
-(to be written).
+The following instructions are for SLES15 SP5.
+The RPM packages can be downloaded from (to be determined).
```
Comment (Bischoff) [inline doc/source/setuphttpd.rst:86]: please put download URL here (from `Virtualisation:feilong`)

same for RHEL, BTW

please check there are no more `(to be determined)` remarks :smile_cat:

**Review by Bischoff — CHANGES_REQUESTED** [review]
see last remarks

File: doc/source/setuphttpd.rst (hunk 5 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst:37]: Sure.

File: doc/source/setuphttpd.rst (hunk 7 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst]: Done

File: doc/source/setuphttpd.rst (hunk 3 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst:34]: Done

File: doc/source/setuphttpd.rst (hunk 5 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst]: Done.

File: doc/source/setuphttpd.rst (hunk 6 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst:38]: Noted.

File: doc/source/setuphttpd.rst (hunk 4 above)
Comment (aazam-gh) [reply doc/source/setuphttpd.rst:95]: For now in the current PR, keep the change as is.
As it is already present in the previous page, rather have the commands about starting/stopping the web server, which can be handled in a new PR.

Comment (Bischoff) [reply doc/source/setuphttpd.rst:95]: OK.

By the way, it seems to me that, when you install from packages, apache2 is automatically started, so you don't need instructions to start it in that specific case.

But maybe I'm starting to overthink it.

Comment (mfriesenegger) [reply doc/source/setuphttpd.rst:95]: I think you are thinking correctly but it does not hurt to restate that apache2 might need to be restarted.

**Comment by mfriesenegger** [comment]
@bjhuangr This is ready to be merged.

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**